core/controllers/email_dashboard_test.py

```diff
@@ -156,7 +156,8 @@ def test_email_dashboard_page(self):
 
         response = self.get_html_response('/emaildashboard')
         self.assertIn(
-            '<title itemprop="name">Email Dashboard - Oppia</title>',
+            '@require(\'../../base-components/header.template.html\', '
+            '{"title": "Email Dashboard - Oppia"})',
```

Comment:
I think it will be best to just check for `{"title": "Email Dashboard - Oppia"}` instead of the whole require statement

Reply:
Updated.